.gitignore

```diff
@@ -4,6 +4,7 @@ __pycache__
 *.pyc
 *~
 .eggs/
+artifacts/address.json
```

Comment:
why was this even in the codebase? Is it supposed to be in the codebase? (Question for Samer, I guess)

Reply:
Btw, I modified an address right there and I wanted to push the modifications `address.json` file appeared as modified and ready to be committed so I tried to put that in `.gitignore`.

Reply:
@mariacarmina please remove this from the commit. Trent is already working on deleting this file altogether.